source/sessions/tests/README.rst

```diff
@@ -78,7 +79,26 @@ the given session is *not* marked dirty::
         arguments:
           session: session0
 
+Snapshot session tests
+======================
+Snapshot sessions tests require server of version 5.0 or higher and 
+replica set or a sharded cluster deployment.
+Default snapshot history window on the server is 5 seconds. Running the test in debug mode, or in any other slow configuration
+may lead to `SnapshotTooOld` errors. Drivers can work around this issue by increasing the server's `minSnapshotHistoryWindowInSeconds` parameter, for example:
+
+.. code:: typescript
+
+    client.admin.command('setParameter', 1, minSnapshotHistoryWindowInSeconds=60)
+
+Prose tests
+```````````
+-  Setting both ``snapshot`` and ``causalConsistency`` is not allowed
+
+    * ``client.startSession(snapshot = true, causalConsistency = true)``
+    * Assert that an error was raised by driver
```

Comment:
Can we add a unified test for this? It could run the client.startSession operation with `snapshot = true, causalConsistency = true` arguments and assert that we raise a client-side error with `isClientError: true`.

Reply:
> and we don't have startSession operation.
The unified test runner allows use to call any method on the client including `startSession`. I don't think it's overkill but we don't need to do it in this PR. I'll open a drivers ticket for it.